cmd/metricsexporter/metricsExporter.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/prometheus/common/promlog"
+	promflag "github.com/prometheus/common/promlog/flag"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	exporterNamespace string
+	exporterAddress   string
+	kubeconfigPath    string
+	kubeAPIServerURL  string
+	verbose           bool
+)
+
+type lvmCollector struct{}
+
+func (lc *lvmCollector) Describe(c chan<- *prometheus.Desc) {}
+func (lc *lvmCollector) Collect(c chan<- prometheus.Metric) {}
+
+func main() {
+	lvmExpFlags := flag.NewFlagSet("lvm-metrics-exporter", flag.ExitOnError)
+	lvmExpFlags.StringVar(&exporterNamespace, "namespace", "lvm-exporter",
+		"set the namespace of the lvm metric exporter")
+	lvmExpFlags.StringVar(&exporterAddress, "address", ":23532",
+		"address on which the metrics exporter should run")
+	lvmExpFlags.StringVar(&kubeconfigPath, "kubeconfig", "", "Path to kubeconfig file")
+	lvmExpFlags.StringVar(&kubeAPIServerURL, "apiserver", "", "API server URL")
+	lvmExpFlags.BoolVar(&verbose, "verbose", false, "provide a verbose output")
+
+	lvmExpFlags.Parse(os.Args[1:])
```

Comment:
linter is failing. Need to check the error.

Reply:
Done